app/src/main/java/com/example/lastfm/NewsAdapter.java

```diff
@@ -35,46 +35,45 @@ public void clearItems(){
 
 
         class NewsViewHolder extends RecyclerView.ViewHolder  {
```

Comment:
rename the class

Reply:
Changed whole class to ArtistAdapter and to ArtistsViewHolder